### ISSUE TITLE
Changing the Column Width

### ISSUE THREAD

**Issue by Nicolaiaiai** [simonbengtsson/jsPDF-AutoTable#273]
Hello! It's me again

How do I change the Column Width of my Table?
My Code looks like this:

                base64Img = base64;
                var pageContent = function (data) {
                    /* Header */
                    if (data.pageCount == 1) {
                        doc.addImage(base64Img, 'png', data.settings.margin.left, 15, 30, 5);
                        doc.setFontSize(10);

                        /* Footer */
                        var str = "Seite " + data.pageCount;
                        doc.setFontSize(10);
                        doc.text(str, 180, doc.internal.pageSize.height - 10);

                        doc.setFontSize(8);
                        doc.text("[TEXT], " + datetime, 100, 16);
                        doc.setFontSize(15);
                        doc.text("[TEXT]" + branche, data.settings.margin.left, 36);
                    }
                    else {
                        var str = "Seite " + data.pageCount;
                        doc.setFontSize(10);
                        doc.text(str, 180, doc.internal.pageSize.height -5);
                    }

                };

                var currentdate = new Date();
                var datetime = currentdate.getDate() + "/" + (currentdate.getMonth() + 1) + "/" + currentdate.getFullYear();
                console.log(datetime);
                var doc = new jsPDF();

                var elem = document.getElementById("tbl1");
                var res = doc.autoTableHtmlToJson(elem);
                doc.autoTable(res.columns, res.data, {
                    addPageContent: pageContent,
                    startY: 43,
                    margin: {horizontal: 7},
                    styles: {overflow: 'linebreak', columnWidth: 'auto'},
                    columnStyles: {0: {columnWidth: '15%'}}
                });
                doc.save("[NAME].pdf");
                return doc;
            });

The first Column in my Table is a Number. If the Number binary (for example '23') the Output looks weird...
It looks like this:
Nr ..
2  ...
3  ...
2  ... 
4  ...

I tried solving the Problem by adding this:

columnStyles: {0: {columnWidth: '15%'}}

but it looks like it won't work.

I hope you understand my Problem.
Have a nice Day

Nicolai

**Comment by simonbengtsson** [comment]
Hello mate! I didn't look to much into the code, but there is no support for column width percentages at this point, you have to specify it like so: `{0: {columnWidth: '100'}}`.

**Comment by Nicolaiaiai** [comment]
It's still not working, even though i specify the columnWidth like you did

If i type 

{0: {columnWidth: 100}}

Is the '100' 100px width or 100% width?

**Comment by simonbengtsson** [comment]
My bad, I updated the code to `100` instead of `'100'`. Try that. It is `mm` by default if you have not set it to something else when you created the jsdpf document. From jspdf docs: 

> Measurement unit to be used when coordinates are specified. One of "pt" (points), "mm" (Default), "cm", "in"

**Comment by Nicolaiaiai** [comment]
Well... It seems to work now but first I used your code like that:
`{0: {columnWidth: '100'}}` (not working)
After you replied to me and said that I should use the quotation marks I did this:
`{0: {columnWidth: 100}}` (working)

Isn't that strange? Did I misunterstood you or did you the mistake?

Anyway, Thanks! Its working!

Have a nice day

**Comment by simonbengtsson** [comment]
Yep you are right. It is supposed to be `100`. `'100'` will not work.

**Comment by agindo** [comment]
Use of deprecated style columnWidth. It is renamed to cellWidth.
